doc/man3/FIPS_mode.pod

```diff
@@ -0,0 +1,65 @@
+=pod
+
+=head1 NAME
+
+FIPS_mode, FIPS_mode_with_ctx, FIPS_mode_set, FIPS_mode_set_with_ctx
+- set or get the global property for fips mode
+
+=head1 SYNOPSIS
+
+ #include <openssl/crypto.h>
+
+ int FIPS_mode_set_with_ctx(OPENSSL_CTX *libctx, int on);
+ int FIPS_mode_with_ctx(OPENSSL_CTX *libctx);
+
+ int FIPS_mode_set(int on);
+ int FIPS_mode(void);
+
+=head1 DESCRIPTION
+
+FIPS_mode_set() enables the global property "fips=true" if I<on> is non zero,
+otherwise the global property is set to "".
+Note that if additional global properties are required then
```

Comment:
Here we talk about "global properties". But the EVP_set_default_properties page talks about "default properties for all
future EVP algorithm fetches, implicit as well as explicit". So "global properties" seems like a new term. I think we need to get some consistency here.

Reply:
global properties was the old term.. It no longer really applies - so changed to default properties instead.